Write announcement and documentation for Account Console v3 (#26318)
keycloak/keycloak: docs/documentation/server_admin/topics/account.adoc

== Account Console

> Any {project_name} user can manage their account through the Account Console. Users can configure their profiles, add two-factor authentication, include identity provider accounts, and oversee device activity.

Reviewer: ```suggestion
{project_name} users can manage their accounts through the Account Console. They can configure their profiles, add two-factor authentication, include identity provider accounts, and oversee device activity.
```

* You need to have the *view-groups* account role for being able to view *Groups* menu.

.View group memberships

> image:images/account-console-groups.png[View group memberships]

Reviewer: ```suggestion
.View group memberships
image:images/account-console-groups.png[View group memberships]
```

Author: @andymunro not sure what has changed here, that same header already exists.
